safenode/src/transfers/error.rs

```diff
@@ -0,0 +1,95 @@
+// Copyright 2023 MaidSafe.net limited.
+//
+// This SAFE Network Software is licensed to you under The General Public License (GPL), version 3.
+// Unless required by applicable law or agreed to in writing, the SAFE Network Software distributed
+// under the GPL Licence is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. Please review the Licences for the specific language governing
+// permissions and limitations relating to use of the SAFE Network Software.
+
+use crate::protocol::types::{address::DbcAddress, fees};
+
+use sn_dbc::{Error as DbcError, SignedSpend, Token};
+
+use serde::{Deserialize, Serialize};
+use std::collections::BTreeSet;
+use thiserror::Error;
+
+/// The type returned by the `sn_routing` message handling methods.
```

Comment:
hmm...  there is no longer `sn_routing` ?

Reply:
Old stuff following along from old repo.. (very old stuff... o_0)